keepassxc-browser/keepassxc-browser.js

```diff
@@ -1231,8 +1231,22 @@ cipObserverHelper.getId = function(target) {
     return target.classList.length === 0 ? target.id : target.classList;
 };
 
-cipObserverHelper.handleObserverAdd = function(target) {
+cipObserverHelper.ignoredElement = function(target) {
+     // Ignore SVG elements
+    if (target.className && (target.nodeName !== 'svg' || target.parentNode.nodeName !== 'svg')) {

```

Comment:
The `target.className` seems unnecessary for this conditional

Reply:
The whole conditional needed to be fixed. Thanks for noticing this.